lib/PlacesProvider.js

```diff
@@ -194,17 +194,25 @@ Links.prototype = {
       limit = HISTORY_RESULTS_LIMIT;
     }
 
-    let sqlQuery =  `SELECT moz_places.url as url, moz_favicons.data as favicon, mime_type as mimeType,
-                            title, frecency, last_visit_date as lastVisitDate,  "history"  as type
+    let sqlQuery =  `SELECT moz_places.url as url,
+                            moz_favicons.data as favicon,
+                            mime_type as mimeType,
+                            moz_places.title,
+                            frecency,
+                            last_visit_date as lastVisitDate,
+                            "history"  as type,
+                            moz_bookmarks.id as bookmark
```

Comment:
I don't mean to say we need to implement it here. Just to see if bookmarks.id would be enough to implement that in the future